functions/index.js

```diff
@@ -439,15 +474,17 @@ const deleteComment = (req, res) => {
 
 // User routes
 app.post("/user", signupUser);
+app.post("/user/guest", guestAccount);
 app.post("/user/login", loginUser);
 app.post("/user/logout", logoutUser);
 app.delete("/user/:username", deleteUser);
 app.put("/user/:username", updateUser);
+app.put("/user/recover", recoverPassword);
```

Comment:
there's no username here, how do you know which is trying to recover the password?